### ISSUE TITLE
Align login and < buttons on the resource page

### ISSUE THREAD

**Issue by analytically** [concourse/concourse#3421]
The resources navigation has a small alignment issue between the login button and the < and > buttons. IMHO visually it makes sense to align the buttons vertically.

![image](https://user-images.githubusercontent.com/254434/53735359-e761d900-3e86-11e9-9cd8-6a7b635fdfdc.png)

**Comment by pivotal-jamie-klassen** [comment]
I agree that it is visually unsettling to see the chevrons be *almost but not quite* the same width as the login button, but while we're here, let's also consider the logged-in case:

Rather have a width fixed to that of *what the login button would be if it were there* in the case when you are logged in, I wonder if the chevrons should always match the width of the user menu (it will [expand with the username to a maximum of 20% of the viewport width](https://github.com/concourse/concourse/blob/master/web/elm/src/TopBar/Styles.elm#L250-L252))? @Lindsayauchin

So rather than this:
![screen shot 2019-03-04 at 9 54 46 am](https://user-images.githubusercontent.com/22056320/53741123-aa381f80-3e63-11e9-8fa4-a1c862ad785a.png)

We would see this:
![screen shot 2019-03-04 at 9 57 53 am](https://user-images.githubusercontent.com/22056320/53741295-0e5ae380-3e64-11e9-8c19-d0e43d6b6eca.png)

In this case I manually added `width: 88px` and `justify-content: center` to each of the chevron containers... The logic in general here might be a bit more delicate as there is no common container element determining width for the user menu and pagination.

**Comment by Lindsayauchin** [comment]
I like aligning the logged-in with the pagination buttons. The misalignment catches your eye and algining them will look more polished. 

![screen shot 2019-03-04 at 10 48 06 am](https://user-images.githubusercontent.com/27746614/53744418-06527200-3e6b-11e9-9012-db2d5e0d1673.png)

I think that the logged-in user name expanding the container works well currently. But, if we expand the buttons to mimic that width we will end up with the buttons moving around on the user. I think we should just leave them as they are and just fix the logged-in state. 
Thoughts?

**Comment by vito** [comment]
This seems like a coincidence tbh. Aligning these two elements together
feels somewhat arbitrary; it's just as likely to be out of alignment with
any other page, e.g. the build trigger button.

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
